### PR TITLE
Added update profile feature

### PR DESCRIPTION
**Pull request by ghost**
@jwall149 @huydinhq Please take a look when you have time. Thank you.

### CODE REVIEW

File: controllers/user.js [hunk 1]
```diff
@@ -110,4 +110,29 @@ module.exports = {
...
       .then(() => ({ status: 'OK', message: 'Your password has been changed.' }));
   }),
+  getProfile(req, res) {
+    res.render('account/profile');
+  },
+  postUpdateProfile: promisifyJSON((req, res) => {
+    const errors = req.validationErrors();
+    if (errors) {
+      req.flash('errors', errors);
+      return res.redirect('/account/profile');
```
Comment (jwall149) [inline controllers/user.js]: This is a ajax call, so res.redirect does not work, better implement redirect at frontend.

File: views/account/profile.pug [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          i.fa.fa-pencil
+          | Edit
+
+    .row.hidden#profile_form
+      .col-md-6.col-md-offset-3
+        form(method='POST' action='/account/profile')
+          input(type='hidden', name='_csrf', value=_csrf)
+          div.form-group
+            label(for='name') Name:
+            input#name.form-control(type='text', placeholder='first and last' name='name', value=user.profile.name)
```
Comment (rr-huy-dinh) [inline views/account/profile.pug]: Upper case first character

File: views/account/profile.pug [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+              | Male
+            label.radio-inline
+              input#optionsRadios(type="radio", name="gender", value="female", checked=user.profile.gender=="female")
+              | Female
+            label.radio-inline
+              input#optionsRadios(type="radio", name="gender", value="other", checked=user.profile.gender=="other")
+              | Other
+          div.form-group
+            label(for='location') Location:
+            input#name.form-control(type='text', placeholder='location' name='location', value=user.profile.location)
```
Comment (rr-huy-dinh) [inline views/account/profile.pug]: Upper case first character

File: views/account/profile.pug [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+          div.form-group
+            label(for='location') Website:
+            input#name.form-control(type='text', placeholder='location' name='website', value=user.profile.website)
```
Comment (rr-huy-dinh) [inline views/account/profile.pug]: Upper case first character

File: controllers/user.js (hunk 1 above)
Comment (ghost) [reply controllers/user.js]: No, I don't use ajax here. I submit a form normally. Do you think we need to use ajax?

**Comment by ghost** [comment]
@jwall149 @huydinhq I revised the code to use ajax call and changed first character of placeholder text to uppercase.

**Pull request merged**